cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -268,6 +268,30 @@ def test_suspend(
 
 @pytest.mark.long_running
 @pytest.mark.uncollectif(lambda provider: provider.type != 'openstack')
+def test_pause(
+        setup_provider_funcscope, provider, testing_instance, soft_assert,
+        verify_vm_running):
+    """ Tests instance unpause
+
+    Metadata:
+        test_flag: power_control, provision
+    """
+    testing_instance.wait_for_vm_state_change(
```

Comment:
Personally I'd probably add a "vm_start" either here, or in the fixture, the fixture probably makes more sense. Even though create_on_provider may start it, I feel we should be explicit.